concepticondata/conceptlists/Bowern-2021-207a.tsv

```diff
@@ -0,0 +1,208 @@
+Bowern-2021-207a-29	29	blow	370	body	Basic Vocabulary	176	BLOW (WITH MOUTH)
+Bowern-2021-207a-30	30	blunt	568	quality	Basic Vocabulary	379	BLUNT
+Bowern-2021-207a-31	31	boil/pimple	372	body	Basic Vocabulary	793	BOIL (OF SKIN)
+Bowern-2021-207a-32	32	bone	373	body	Basic Vocabulary	1394	BONE
+Bowern-2021-207a-33	33	boomerang	569	manufacture	Basic Vocabulary	135	BOOMERANG
+Bowern-2021-207a-34	34	breast	376	body	Basic Vocabulary	1402	BREAST
+Bowern-2021-207a-35	35	breathe	377	body	Basic Vocabulary	1407	BREATHE
+Bowern-2021-207a-36	36	burn	378	environment	Basic Vocabulary	2102	BURN
+Bowern-2021-207a-37	37	chew	379	body	Basic Vocabulary	321	CHEW
+Bowern-2021-207a-38	38	child	380	human	Basic Vocabulary	2099	CHILD
+Bowern-2021-207a-39	39	climb	381	motion	Basic Vocabulary	1102	CLIMB
+Bowern-2021-207a-40	40	cloud	382	environment	Basic Vocabulary	1489	CLOUD
+Bowern-2021-207a-41	41	cold	383	quality	Basic Vocabulary	1287	COLD
+Bowern-2021-207a-42	42	come	384	motion	Basic Vocabulary	1446	COME
+Bowern-2021-207a-43	43	cook	385	impact	Basic Vocabulary	1100	COOK (SOMETHING)
+Bowern-2021-207a-44	44	correct/true	386	quality	Basic Vocabulary	1725	CORRECT (RIGHT)
+Bowern-2021-207a-45	45	count	387	mental	Basic Vocabulary	1420	COUNT
+Bowern-2021-207a-46	46	cry	388	mental	Basic Vocabulary	1839	CRY
+Bowern-2021-207a-47	47	cut	570	impact	Basic Vocabulary	1432	CUT
+Bowern-2021-207a-48	48	cut/hack	389	impact	Basic Vocabulary	155	CHOP
+Bowern-2021-207a-49	49	day	390	time	Basic Vocabulary	1225	DAY (NOT NIGHT)
+Bowern-2021-207a-50	50	die/be dead	391	state	Basic Vocabulary	2101	BE DEAD OR DIE
+Bowern-2021-207a-51	51	dig	392	impact	Basic Vocabulary	1418	DIG
+Bowern-2021-207a-52	52	dingo/wolf	394	fauna	Basic Vocabulary	2461	WOLF-LIKE ANIMAL
+Bowern-2021-207a-53	53	dirty	395	quality	Basic Vocabulary	1230	DIRTY
+Bowern-2021-207a-54	54	dream	397	mental	Basic Vocabulary	1920	DREAM (SOMETHING)
+Bowern-2021-207a-55	55	drink	398	body	Basic Vocabulary	1401	DRINK
+Bowern-2021-207a-56	56	dry	399	quality	Basic Vocabulary	1398	DRY
+Bowern-2021-207a-57	57	dull/blunt	400	quality	Basic Vocabulary	379	BLUNT
+Bowern-2021-207a-58	58	dust	401	environment	Basic Vocabulary	2	DUST
+Bowern-2021-207a-59	59	ear	402	body	Basic Vocabulary	1247	EAR
+Bowern-2021-207a-60	60	earth/soil	403	environment	Basic Vocabulary	1228	EARTH (SOIL)
+Bowern-2021-207a-61	61	eat	404	body	Basic Vocabulary	1336	EAT
+Bowern-2021-207a-62	62	egg	405	fauna	Basic Vocabulary	744	EGG
+Bowern-2021-207a-63	63	eye	406	body	Basic Vocabulary	1248	EYE
+Bowern-2021-207a-64	64	faeces	407	body	Basic Vocabulary	675	FAECES (EXCREMENT)
+Bowern-2021-207a-65	65	fall	408	motion	Basic Vocabulary	1280	FALL
+Bowern-2021-207a-66	66	far	409	quality	Basic Vocabulary	1406	FAR
+Bowern-2021-207a-67	67	fat/grease	410	body	Basic Vocabulary	323	FAT (ORGANIC SUBSTANCE)
+Bowern-2021-207a-68	68	father	411	kinship	Basic Vocabulary	1217	FATHER
+Bowern-2021-207a-69	69	fear	412	mental	Basic Vocabulary	781	FEAR (FRIGHT)
```

Comment:
In other Bowern lists this is mapped to FEAR (BE AFRAID).

Reply:
The Spanish gloss indicated only a noun (similar comments will appear a number of times now, so this is maybe a general question of what to make of Spanish/Portuguese glosses if available?)

Reply:
Ah ok, this is a bit tricky. I guess for Australian and North American native languages the English glosses were used, and for the South American the Spanish glosses. I would therefore follow the English glosses to keep it consistent with other Bowern lists. But @chrzyki or @LinguList What do you think?

Reply:
Yes, I agree with @AnnikaTjuka here, keeping it consistent is more important I'd say.